### DISCUSSION TITLE
Help. I don't understand Client ID, Client Secret

### DISCUSSION THREAD

**Discussion by mifotaku** [KRTirtho/spotube#49] [Q&A]
just downloaded the app. Now I don't understand how to login

**Comment by KRTirtho** [comment]
Did you read the README carefully? It's right here https://github.com/KRTirtho/spotube#optional-configurations

**Reply by mifotaku** [reply]
After making Client id. When I submit from the app, It says "INVALID_CLIENT: Invalid redirect URI"

**Reply by KRTirtho** [reply]
You probably didn't add `http://localhost:4304/auth/spotify/callback` in the Redirect URIs
![image](https://user-images.githubusercontent.com/61944859/161703037-2009b9ad-cdc8-4dfd-b74f-96d354f6ab08.png)

**Comment by senolfndk** [comment]
😊👍
![Ekran görüntüsü 2022-05-12 115535](https://user-images.githubusercontent.com/52818836/168032617-1e46e962-f8ac-444d-8020-dbfdb126611c.png)
